### PR TITLE
Show loading message, structure in way that adding dynamic informatio…

### PR DESCRIPTION
**Pull request by qingweibinary**
…n when reconnecting is trivial, for example reconnecting times

### CODE REVIEW

File: src/_store/AppInfoProxy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import LoadingView from '../loading-view/LoadingView';
+
+@connect(state => ({ appInfo: state.appInfo }))
+export default class AppInfoProxy extends Component {
+    static propTypes = {
```
Comment (qingweibinary) [inline src/_store/AppInfoProxy.js]: Not sure if this is a poor name, suggestion are welcome
We can later tap into this component to provide additional information if we necessary ~

I kinda over-engineered it as I was thinking to show how many time they failed, and possibly ask them to check their connection if failure exceeded some threshold, but then found out ws failure was not handled well from the library, so I'll leave it here at the moment

Comment (borisyankov) [inline src/_store/AppInfoProxy.js]: I don't think it matters how many times did we try to reconnect.
But it matters how long was it till we had a connection to the server.

So:
1. Silent retries.
2. After, say 5 seconds of being disconnected, we tell them they are no longer online (5 sec might be too long, we need to test)

**Comment by borisyankov** [comment]
Looks ok. Good to merge?

File: src/_store/AppInfoProxy.js (hunk 1 above)
Comment (qingweibinary) [inline src/_store/AppInfoProxy.js]: I think 5 is ok if there's no connection at all ~

**Comment by qingweibinary** [comment]
OK, please merge ~ and create a card for the silent retry with interval ~

**Pull request merged**